mysql-test/suite/rocksdb_rpl/r/rpl_rocksdb_extra_col_slave.result

```diff
@@ -10,6 +10,8 @@ call mtr.add_suppression("The slave coordinator and worker threads are stopped,
 **** Diff Table Def Start ****
 *** On Slave ***
 STOP SLAVE;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained.